### PR TITLE
First pass at agile process doc

### PR DESCRIPTION
**Pull request by petermarks-gsa**

### CODE REVIEW

File: docs/agile-process.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+### Overview
+
+There is a [high-level roadmap](https://app.mural.co/t/gsa6/m/gsa6/1646336450605/6a77a27a287989f91e1c0dfbcf9be4dec6e49393?sender=ua4d37dfba3f1e69e09078790) that the product and design team distills down to actionable tasks at the beginning of each sprint. Design, engineering and product work is tracked on the [FAC Task Tracking board](https://github.com/GSA-TTS/FAC/projects/1) in Github. "Story" issues can be created to define a high-level piece of functionality, but linked subtasks should be created for discrete pieces of work across disciplines, each not exceeding one day in anticipated time. There is no estimation process for tickets, but they should be groomed to be in approximately comparable sizes. The team works in two-week sprints starting on Tuesdays.
+
+### Board workflow
+
+-   **Backlog** - Not currently selected for the current sprint
+
+-   **Available** - Selected for the current sprint and is currently available to be picked up.
```
Comment (petermarks-gsa) [inline docs/agile-process.md:12]: Suggesting we rename “To do” to “Available” to communicate these issues can be picked up

File: docs/agile-process.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+-   **In progress** - Someone is actively working on this
+
+-   **Blocked** - If you put an issue here, please explain why it is stuck.
```
Comment (petermarks-gsa) [inline docs/agile-process.md:16]: Suggesting we rename from “Blocked/waiting” to “blocked” to clarify something in review versus “stuck”

File: docs/agile-process.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+-   **In review** - There is code review or QA happening on this feature
```
Comment (petermarks-gsa) [inline docs/agile-process.md:18]: New status I’m proposing

File: docs/agile-process.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+-   **Done (Sprint X)** - At the time of writing, "done" for a task means it passed code review and was merged/deployed. "Done" for a story means it's been deployed and has had its acceptance criteria created. Sprint-specific "done" columns are used to track exactly what was completed in a current sprint for velocity / reporting purposes.
```
Comment (petermarks-gsa) [inline docs/agile-process.md:20]: This is a proposal to have multiple “Done” columns at the end of the board.

**Comment by petermarks-gsa** [comment]
It came to my attention that we already have an effort to document this in the wiki, specifically here: 

https://github.com/GSA-TTS/FAC/wiki/story-lifecycle

I've started to square that doc with @jadudm's work [here](https://docs.google.com/document/d/1A-cceurTAvoui0vZRkJ_331v70ZA0mgK7Kn9Dv84zYc/edit#). 

How we make changes to wiki is not yet clear (does someone own it? how are changes made?), but I've taken on the [task](https://github.com/GSA-TTS/FAC/issues/13) to clarify that.

**Review by jadudm — APPROVED** [review]
This looks good to me. Thank you!

I'd personally say we can go ahead and merge, given the general "thumbs-up" I'm seeing, and we can always revise/tweak based on lived experience.

**Comment by petermarks-gsa** [comment]
Merged and updated the wiki: https://github.com/GSA-TTS/FAC/wiki/Agile-Process

**Pull request merged**